lib/install/development.rb

```diff
@@ -0,0 +1,152 @@
+def write_redis_recommendation(development, lines, index)
+  gemfile = Rails.root.join("Gemfile")
+  redis_pattern = /gem ['"]redis-session-store['"]/
+
+  # add redis-session-store to Gemfile, but comment it out
+  if !File.read(gemfile).match?(redis_pattern)
+    append_file gemfile, "\n# Use Redis for session storage", verbose: false
+    gem "redis-session-store", "0.11.4"
+    comment_lines gemfile, redis_pattern
+  end
+
+  # provide a recommendation for using redis-session-store, including commented source code
+  if !lines.index { |line| line.include?("StimulusReflex does not support :cookie_store") }
+    lines.insert index + 1, <<RUBY
+
+  # StimulusReflex does not support :cookie_store, and we recommend switching to Redis.
+  # https://docs.stimulusreflex.com/appendices/deployment#use-redis-as-your-cache-store
+
+  # To use `redis-session-store`, make sure to add it to your Gemfile and run `bundle install`.
+
+  # config.session_store :redis_session_store, {
+  #   serializer: :json,
+  #   on_redis_down: ->(*a) { logger.error("Redis down! \#{a.inspect}") },
+  #   redis: {
+  #     expire_after: 120.minutes,
+  #     key_prefix: "session:",
+  #     url: ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" }
+  #   }
+  # }
```

Comment:
```suggestion
  # config.session_store :redis_session_store
  #   serializer: :json,
  #   on_redis_down: ->(*a) { logger.error("Redis down! \#{a.inspect}") },
  #   redis: {
  #     expire_after: 120.minutes,
  #     key_prefix: "session:",
  #     url: ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" }
  #   }
```

Reply:
Nice catch. What version of Ruby are you using?

Reply:
Dropping the (apparently superfluous) extra `{` `}` pair seems fine in 2.7 as well!

Reply:
Sorry for the late answer. I used Ruby 3.1.2.

Reply:
No worries! Lemme know if the latest works for you.

Reply:
I still get the same errors with the latest changes 🤔